src/file-upload/file-upload.scss

```diff
@@ -39,18 +40,39 @@
       z-index: -1;
     }
 
-    > .iui-icon {
+    > svg {
       @include iui-icons-3xl;
-
-      + span,
-      + label {
-        margin-top: $iui-baseline;
-      }
+      margin-top: $iui-baseline;
+      margin-bottom: $iui-baseline;
     }
 
     &:only-child {
       visibility: visible;
       position: relative;
+      flex-direction: row;
```

Comment:
Yeah I think all the new styling should go under a new class so that it doesn't mess with other styling.

Without a separate class, the flex-direction makes it hard to display "mixed content" (provided by us and by user)
![image](https://user-images.githubusercontent.com/9084735/116932667-8c77c800-ac30-11eb-874a-20f40fb7de51.png)

(the link should go below the label, not to the right)

Reply:
This has been partly fixed by adding `.iui-label` class (which contains main `label.iui-anchor` and secondary `div` for extra text).

If user were to add anything outside the `.iui-label`, it would still go to the right. I debated adding a new `.iui-template` class but decided not to.